<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Alain malvoisin</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>684</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>710</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CAN bus et NMEA 2000 en temps réél</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <selectedon>:/icones/ps2.png</selectedon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="lab_octet">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(254, 255, 221);</string>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>                            Numéro de ligne
 PGN                 Priorité   Source    Destination

 Vient des datas :
 Significations
 Valeurs
 Table</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="3" rowspan="6" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Table d'affichage des données reçues du bus CAN.</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QTableView" name="table_can">
           <property name="minimumSize">
            <size>
             <width>310</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(85, 255, 255);</string>
           </property>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>24</number>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>30</number>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 127);</string>
      </property>
      <widget class="QLineEdit" name="line_nmea">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>55</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_table">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>35</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>55</x>
         <y>60</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>NMEA 2000 en CSV</string>
       </property>
       <property name="buddy">
        <cstring>line_nmea</cstring>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>55</x>
         <y>35</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Buffer de la Table CAN</string>
       </property>
       <property name="buddy">
        <cstring>line_table</cstring>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>10</y>
         <width>321</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Nombre de trame dans la table et sur NMEA 2000</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QWidget" name="widget_5" native="true">
      <widget class="QWidget" name="widget_6" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-5</y>
         <width>331</width>
         <height>100</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <widget class="QCheckBox" name="check_file">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>55</y>
          <width>246</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Enregistrer le bus CAN sur le fichier</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="check_nmea">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>75</y>
          <width>226</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Lancer le NMEA 2000 en temps réel</string>
        </property>
       </widget>
       <widget class="QLabel" name="lab_file">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>5</y>
          <width>321</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Fichier texte du bus CAN ------------------------</string>
        </property>
       </widget>
       <widget class="QLabel" name="lab_csv">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>25</y>
          <width>321</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Fichier CSV en NMEA 2000 ---------------------</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="3" column="0" rowspan="2" colspan="2">
     <widget class="QWidget" name="widget_3" native="true">
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>10</y>
         <width>341</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../HUAHINE/images/regate.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>5</y>
         <width>346</width>
         <height>106</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>images/regate.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>205</x>
         <y>75</y>
         <width>55</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>images/VoilierImage2.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>75</y>
         <width>55</width>
         <height>56</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>images/Spi.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>115</x>
         <y>110</y>
         <width>55</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>images/Image3.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QWidget" name="widget_4" native="true">
      <widget class="QCheckBox" name="check_buffer">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>45</y>
         <width>16</width>
         <height>36</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>135</x>
         <y>55</y>
         <width>191</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Remplir la table en temps réel-&gt;</string>
       </property>
       <property name="buddy">
        <cstring>check_buffer</cstring>
       </property>
      </widget>
      <widget class="QLabel" name="lab_connection">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>316</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 85, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>742</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionOuvrir"/>
    <addaction name="actionVoir"/>
    <addaction name="actionImporter"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuTable">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionAide"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuBus_CAN">
    <property name="title">
     <string>Bus CAN</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="actionRead"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionStatus"/>
   </widget>
   <widget class="QMenu" name="menuCarte">
    <property name="title">
     <string>Carte</string>
    </property>
    <addaction name="actionMap"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuBus_CAN"/>
   <addaction name="menuCarte"/>
   <addaction name="menuTable"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOuvrir"/>
   <addaction name="actionVoir"/>
   <addaction name="actionImporter"/>
   <addaction name="actionExport"/>
   <addaction name="separator"/>
   <addaction name="actionOpen"/>
   <addaction name="actionClose"/>
   <addaction name="actionRead"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionMap"/>
   <addaction name="separator"/>
   <addaction name="actionStatus"/>
   <addaction name="separator"/>
   <addaction name="actionAide"/>
   <addaction name="separator"/>
   <addaction name="actionQuitter"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionQuitter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icones/Quitter.png</normaloff>icones/Quitter.png</iconset>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="toolTip">
    <string>Quitter l'application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionImporter">
   <property name="icon">
    <iconset>
     <normaloff>icones/Table.ico</normaloff>icones/Table.ico</iconset>
   </property>
   <property name="text">
    <string>Importer le fichier texte CAN sur la table</string>
   </property>
   <property name="toolTip">
    <string>Importer le fichier CAN dans la table</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionVoir">
   <property name="icon">
    <iconset>
     <normaloff>icones/Reading.png</normaloff>icones/Reading.png</iconset>
   </property>
   <property name="text">
    <string>Lire le fichier texte</string>
   </property>
   <property name="toolTip">
    <string>Lire le fichier du bus CAN</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionOuvrir">
   <property name="icon">
    <iconset>
     <normaloff>icones/Ouvir.gif</normaloff>icones/Ouvir.gif</iconset>
   </property>
   <property name="text">
    <string>Ouvrir un fichier texte</string>
   </property>
   <property name="toolTip">
    <string>Ouvrir un fichier texte</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>icones/OpenMal.gif</normaloff>icones/OpenMal.gif</iconset>
   </property>
   <property name="text">
    <string>Ouvre l'dapatateur</string>
   </property>
   <property name="toolTip">
    <string>Ouvre l'adaptateur CAN</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset>
     <normaloff>icones/CloseMal.gif</normaloff>icones/CloseMal.gif</iconset>
   </property>
   <property name="text">
    <string>Ferme l'adaptateur</string>
   </property>
   <property name="toolTip">
    <string>Ferme l'adaptateur CAN</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionRead">
   <property name="icon">
    <iconset>
     <normaloff>icones/Talk.gif</normaloff>icones/Talk.gif</iconset>
   </property>
   <property name="text">
    <string>Lecture du  bus CAN</string>
   </property>
   <property name="toolTip">
    <string>Lecture sur le bus CAN en temps réel</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionStatus">
   <property name="icon">
    <iconset>
     <normaloff>icones/Status.png</normaloff>icones/Status.png</iconset>
   </property>
   <property name="text">
    <string>Lire le Status</string>
   </property>
   <property name="toolTip">
    <string>Lire le Status du bus CAN</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset>
     <normaloff>icones/STOP.gif</normaloff>icones/STOP.gif</iconset>
   </property>
   <property name="text">
    <string>STOP la lecture</string>
   </property>
   <property name="toolTip">
    <string>STOP la lecture CAN en temps réel</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>icones/ps2.png</normaloff>icones/ps2.png</iconset>
   </property>
   <property name="text">
    <string>A propos</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset>
     <normaloff>icones/csv.png</normaloff>icones/csv.png</iconset>
   </property>
   <property name="text">
    <string>Exporter en CSV le NMEA 2000</string>
   </property>
   <property name="toolTip">
    <string>Exporter NMEA 2000 vers un CSV</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionMap">
   <property name="icon">
    <iconset>
     <normaloff>icones/VoilierImage.png</normaloff>icones/VoilierImage.png</iconset>
   </property>
   <property name="text">
    <string>Voir la carte</string>
   </property>
   <property name="toolTip">
    <string>Voir le voilier sur la carte</string>
   </property>
  </action>
  <action name="actionAide">
   <property name="icon">
    <iconset>
     <normaloff>icones/aides.png</normaloff>icones/aides.png</iconset>
   </property>
   <property name="text">
    <string>Aide</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
